src/util/extend.ts

```diff
@@ -1,4 +1,4 @@
-export default function extendBy(output: any, ...inputs: Array<any>) {
+export function extendBy(output: any, ...inputs: Array<any>) {
```

Comment:
I wonder if this can be removed now and we can use some typescript functionality

Reply:
Keep it for now then, it's out of scope for this PR...